docs/release-notes.md

```diff
@@ -5,7 +5,7 @@ blocks)](https://github.com/kubernetes/kubernetes/blob/master/.github/PULL_REQUE
 labels](https://github.com/kubernetes/community/blob/master/contributors/guide/release-notes.md) with compliance across master and release- branches using [test-infra/prow](https://github.com/kubernetes/test-infra/tree/master/prow/plugins/releasenote)
 on the main Kubernetes repository (and future repositories later).
 
-Releases are built and published by the anago tool in [this repo](https://github.com/kubernetes/release) with the release notes published in [kubernetes/CHANGELOG-x.y.md files](https://github.com/kubernetes/kubernetes).
+Releases are built and published by the anago tool in [this repo](https://github.com/kubernetes/release) with the release notes published in [kubernetes/CHANGELOG/CHANGELOG-x.y.md files](https://github.com/kubernetes/kubernetes/tree/master/CHANGELOG).
```

Comment:
Vanity URL here as well. You'll need to drop `/tree/master`.

Reply:
Fixed that as well.